CoSA.py

```diff
@@ -73,6 +74,7 @@ def __init__(self):
         self.strfiles = None
         self.verbosity = 1
         self.simulate = False
+        self.sim_no_unroll = False
```

Comment:
I would see the simulation without unrolling as a parameter of simulation, instead of a separate analysis